build.gradle

```diff
@@ -131,3 +135,10 @@ shadowJar {
 }
 
 assemble.dependsOn(shadowJar)
+
+// Invoke arbitrary classes interactively. Example:
+// ./gradlew runJob -P demoJob=io.crate.flink.demo.SimpleTableApiJob
+task runJob(type:JavaExec) {
+    classpath = sourceSets.main.runtimeClasspath
+    main = project.hasProperty("demoJob") ? project.getProperty("demoJob") : null
+}
```

Comment:
The fallback is needed, otherwise, launching the root project will complain like:
```
Could not get unknown property 'demoJob' for root project 'cratedb-flink-jobs' of type org.gradle.api.Project.
```